pkg/controller/backupscheduled/backupscheduled_controller.go

```diff
@@ -104,8 +106,12 @@ type ReconcileBackupScheduled struct {
 // +kubebuilder:rbac:groups=extension.shepherd,resources=backupscheduleds/finalizers,verbs=get;list;watch;create;update;patch;delete
 func (r *ReconcileBackupScheduled) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	r.log = logger.New(ControllerName, request.Namespace, request.Name)
-	r.log.SetLevel("debug")
-	r.log.Info("Starting reconcile loop")
+	r.log.SetLevel("info")
+	if os.Getenv("DEBUG") == "debug" {
+		r.log.SetLevel("debug")
```

Comment:
Nice, had wondered how to do this sort of thing in here, but hadn't actually found the logging yet.